lib/vapor/config/file.ex

```diff
@@ -26,19 +26,40 @@ defmodule Vapor.Config.File do
     end
   end
 
+  def pathify_keys({:ok, map}) when is_map(map) do
```

Comment:
I woke up thinking about this problem this morning. I realized I could put the path-ification of keys right before we load the the configs into the data store. I ended up liking this a lot better because 1) the function doesn't add to the interface and can be private, and 2) additional config backends won't need to _remember_ to call it--if we get a nested structure from a backend, the store ensures that its normalized before it goes into ETS.